_includes/pbc/all/install-features/202304.0/install-the-shipment-feature.md

```diff
@@ -149,7 +184,206 @@ Make sure that the configured data has been added to the `spy_glossary_key` and
+{
+    /**
+     * @return array<int|string, \Spryker\Zed\PublisherExtension\Dependency\Plugin\PublisherPluginInterface>|array<string, array<int|string, \Spryker\Zed\PublisherExtension\Dependency\Plugin\PublisherPluginInterface>>
+     */
+    protected function getPublisherPlugins(): array
+    {
+        return array_merge(
+            $this->getShipmentTypeStoragePlugins(),
+        );
+    }
+
+    /**
+     * @return lsit<\Spryker\Zed\PublisherExtension\Dependency\Plugin\PublisherTriggerPluginInterface>
+     */
+    protected function getPublisherTriggerPlugins(): array
+    {
+        return [
+            new ShipmentTypePublisherTriggerPlugin(),
+        ];
+    }
+
+    /**
+     * @return list<\Spryker\Zed\PublisherExtension\Dependency\Plugin\PublisherPluginInterface>
+     */
+    protected function getShipmentTypeStoragePlugins(): array
+    {
+        return [
+            new ShipmentTypeWriterPublisherPlugin(),
+            new ShipmentTypeStoreWriterPublisherPlugin(),
+        ];
+    }
+}
+```
+
+5. Set up synchronization plugins:
+
+| PLUGIN                                              | SPECIFICATION                                                            | PREREQUISITES | NAMESPACE                                                            |
+|-----------------------------------------------------|--------------------------------------------------------------------------|---------------|----------------------------------------------------------------------|
+| ShipmentTypeSynchronizationDataBulkRepositoryPlugin | Allows synchronizing the shipment type storage table content into Redis. |               | Spryker\Zed\ShipmentTypeStorage\Communication\Plugin\Synchronization |
+
+**src/Pyz/Zed/Synchronization/SynchronizationDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Synchronization;
+
+use Spryker\Zed\ShipmentTypeStorage\Communication\Plugin\Synchronization\ShipmentTypeSynchronizationDataBulkRepositoryPlugin;
+use Spryker\Zed\Synchronization\SynchronizationDependencyProvider as SprykerSynchronizationDependencyProvider;
+
+class SynchronizationDependencyProvider extends SprykerSynchronizationDependencyProvider
+{
+    /**
+     * @return list<\Spryker\Zed\SynchronizationExtension\Dependency\Plugin\SynchronizationDataPluginInterface>
+     */
+    protected function getSynchronizationDataPlugins(): array
+    {
+        return [
+            new ShipmentTypeSynchronizationDataBulkRepositoryPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the `shipment-type` trigger plugin works correctly:
+
+1. Fill the `spy_shipment_type`, `spy_shipment_type_store` tables with data.
+2. Run the `console publish:trigger-events -r shipment_type` command.
+3. Make sure that the `spy_shipment_type_storage` table has been filled with respective data.
+4. Make sure that, in your system, storage entries are displayed with `kv:shipment_type:{store}:{shipment_type_id}` mask.
```

Comment:
I think it's not necessary because explanations will not bring any additional value. These are standard placeholders for search/storage modules so they are clear for other developers.